### ISSUE TITLE
Planetary API over quota

### ISSUE THREAD

**Issue by Rabrg** [nasa/api-docs#61]
The planetary API has been going over quota halfway through the day for the past two days.

**Comment by jasonduley** [comment]
yes.  we are aware of the issue and are working to resolve it.  thank you.

**Comment by benjaminboruff** [comment]
+1

**Comment by The-Penultimate-Defenestrator** [comment]
Any update? This is affecting one of my projects, even if I register a brand new API key, I get an "over quota" error on my first request

**Comment by The-Penultimate-Defenestrator** [comment]
Possibly related to the deprecation of the Google Earth API? This took effect December 12 (6 days ago) and the URLs returned by the planetary API appear to point to a google site.

**Comment by benjaminboruff** [comment]
OK, I think there is something seriously wrong with this rate-limit issue. There needs to be a more specific rate limit guideline, like x-reqs per second! There is no way to get 1000 reqs per hour when I can't even get 300 reqs in a few minutes. 

If I slow down my reqs using a timer (setTimeout in Node.js) such that my first req occurs in 500 ms, then the second in 1 sec, the third in 1.5 sec, then 2 sec, etc; I can avoid the over the rate limit  issue, but then get hit with error code 500, "Internal Service Error" after about 200 reqs; the 200th req coming 100 secs after the first req - over a minute and a half. According to the header info, I'm not even close to hitting my hourly limit! But, errors nonetheless.

So it looks like the server/service cannot take very much of a load at all.

We live in an async world. Telling a developer that they have a 1000 reqs per hour limit means the developer **_will_** fire off 1000 reqs all at the same time and expect to get back 1000 successful JSON strings of real/good data. That means 1000 requests almost simultaneously! And that's just one dev!

Looks like the backend needs an overhaul.

**Comment by benjaminboruff** [comment]
Wow. Even with a 10 sec interval between requests, only slightly over 50% of them return data **_without_** an error code 500, Internal Service Error.

**Comment by The-Penultimate-Defenestrator** [comment]
I'd personally try to get into Google's earth engine API. I think this is
what powers NASA's API. Check out https://earthengine.google.com/

**Comment by benjaminboruff** [comment]
I'll take a look! I was really hoping for space data, though ... :(

-------------------------------------------------
{
  "name": "Benjamin Boruff",
  "title": "MEAN Developer",
  "url": "http://benjaminboruff.github.io",
  "email": "benboruff@gmail.com",
  "twitter": "@BenjaminBoruff"
}

**Comment by brianthomas** [comment]
Closing this as it is a deployment, not code issue which we can fix.
